<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCustomSine</class>
 <widget class="QDialog" name="DialogCustomSine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Insert sine wave</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QGridLayout" name="gridLayout">
       <item row="6" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Time (seconds):</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Frequency (Hz):</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="KillerDoubleSpinBox" name="doubleSpinBoxSampleRate">
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="maximum">
          <double>99999999999999991433150857216.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBoxPhase">
         <property name="suffix">
          <string>Â°</string>
         </property>
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="maximum">
          <double>360.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Samples:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Sample Rate:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Phase:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="KillerDoubleSpinBox" name="doubleSpinBoxFrequency">
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="maximum">
          <double>9999999999999999931398190359470212947659194368.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Amplitude:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="1">
        <widget class="KillerDoubleSpinBox" name="doubleSpinBoxNSamples">
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="maximum">
          <double>9999999999999999455752309870428160.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="lineEditTime"/>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBoxAmplitude">
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.001000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="ZoomableGraphicView" name="graphicsViewSineWave">
      <property name="renderHints">
       <set>QPainter::Antialiasing|QPainter::HighQualityAntialiasing|QPainter::TextAntialiasing</set>
      </property>
      <property name="resizeAnchor">
       <enum>QGraphicsView::NoAnchor</enum>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ZoomableGraphicView</class>
   <extends>QGraphicsView</extends>
   <header>urh.ui.views.ZoomableGraphicView.h</header>
  </customwidget>
  <customwidget>
   <class>KillerDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>urh.ui.KillerDoubleSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>doubleSpinBoxAmplitude</tabstop>
  <tabstop>doubleSpinBoxFrequency</tabstop>
  <tabstop>doubleSpinBoxPhase</tabstop>
  <tabstop>doubleSpinBoxSampleRate</tabstop>
  <tabstop>doubleSpinBoxNSamples</tabstop>
  <tabstop>lineEditTime</tabstop>
  <tabstop>graphicsViewSineWave</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
